lms/models/event.py

```diff
@@ -17,6 +17,7 @@ class Type(str, Enum):
         EDITED_ASSIGNMENT = "edited_assignment"
         SUBMISSION = "submission"
         GRADE = "grade"
+        ERROR_CODE = "error_code"
```

Comment:
I reckon this is not just "ERROR", the interesting part here is that we know is known error, that we have a code for and that we send that to the FE.